community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/MatchAggregationsBackedByCountStoreAcceptanceTest.scala

```diff
@@ -260,45 +260,6 @@ class MatchAggregationsBackedByCountStoreAcceptanceTest extends ExecutionEngineF
       })
   }
 
-  test("counts relationships with type, any direction and labeled source node without using count store") {
```

Comment:
These tests should remain, but change the test to assert that the count store is no longer used.